src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -153,47 +231,9 @@ export class CodeModelCliImpl implements CodeModelAz
         return this.session.model.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].language['az'].name;
     }
 
-    public SelectFirstExample(): boolean
-    {
-        //this.session.message({Channel:Channel.Warning, Text:serialize(this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].extensions['x-ms-examples'])});
-        let example = this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].extensions['x-ms-examples'];
-        if(example && example.length > 0) {
-            this.currentExampleIndex = 0;
-            return true;
-        } else {
-            this.currentExampleIndex = -1;
-            return false;
-        }
-        
-    }
-
-    public SelectNextExample(): boolean
-    {
-        let example = this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].extensions['x-ms-examples'];
-        if(example && this.currentExampleIndex < example.length - 1) {
-            this.currentExampleIndex++;
-            return true;
-        } else {
-            this.currentExampleIndex = -1;
-            return false;
-        }
-    }
-
-    public get Example_Body(): string[]
-    {
-        // TBD
-        return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].extensions['x-ms-examples'][this.currentExampleIndex].key();
-    }
-
-    public get Example_Params(): any
-    {
-        // TBD
-        return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].extensions['x-ms-examples'][this.currentExampleIndex].value().parameters;
-    }
-
-    public get Example_Title(): string
```

Comment:
we don't need examples for now?

Reply:
just moved them to the end of file, and we need to finish example processor before we can use them